OCPBUGS#44833: Documented the known issue for the 4.18 client used wi… (#88159)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

* A regression in the behaviour of `libreswan` caused some nodes with IPsec enabled to lose communication with pods on other nodes in the same cluster. To resolve this issue, consider disabling IPsec for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-43713[*OCPBUGS-43713*])

> * If you attempted to log in to a {product-title} 4.17 server by using the {oc-first} that you installed as part of the {product-title} {product-version}, a warning message such as the following example outputs in your terminal:

Reviewer: ```suggestion
* If you attempted to log in to a {product-title} 4.17 server by using the {oc-first} that you installed as part of {product-title} {product-version}, you see the following warning message in your terminal:
```

+

> [source,termainl]

Reviewer: ```suggestion
[source,terminal]
```

- A typo

----
Warning: unknown field "metadata"
You don't have any projects. You can try to create a new project, by running

    oc new-project <projectname>
----
+

> This warning message is a known issue, but the message does not indicate any functionality issues with {product-title} functionality. You can safely ignore the message and continue to use {product-title} as intended. (link:https://issues.redhat.com/browse/OCPBUGS-44833[*OCPBUGS-44833*])

Reviewer: ```suggestion
This warning message is a known issue but does not indicate any functionality issues with {product-title}. You can safely ignore the warning and continue to use {product-title} as intended. (link:https://issues.redhat.com/browse/OCPBUGS-44833[*OCPBUGS-44833*])
```
